blogSchema.sql

```diff
@@ -0,0 +1,16 @@
+-- create table statements
+-- A blog, with `users` and `posts`
+
+create table users (
+    id serial primary key,
+    email_address text
+);
+
+create table posts (
+    id serial primary key,
+    date timestamp,
```

Comment:
Great! Just one thing - you'll want to use a different name for this field, as `date` is a reserved word in SQL: https://www.postgresql.org/docs/current/static/datatype-datetime.html

Maybe something like `post_date`?